token/core/fabtoken/deserializer.go

```diff
@@ -55,5 +58,18 @@ func NewEnrollmentIDDeserializer() *enrollmentService {
 }
 
 func (e *enrollmentService) GetEnrollmentID(auditInfo []byte) (string, error) {
+	if len(auditInfo) == 0 {
+		return "", nil
+	}
+
+	// Try to unmarshal it as ScriptInfo
+	si := &interop.ScriptInfo{}
+	err := json.Unmarshal(auditInfo, si)
```

Comment:
what about using our json marshaller in `common`?

Reply:
ok
but why?

Reply:
ah! I had the `common` package in mind we have in FSC. However, this is part of the integration tests. Not applicable at this point.